targets/include/ublksrv_tgt.h

```diff
@@ -117,15 +117,6 @@ static inline unsigned short ublk_unique_tag_to_tag(unsigned int unique_tag)
         return unique_tag & UBLK_UNIQUE_TAG_MASK;
 }
 
-static inline bool ublk_param_is_valid(const struct ublk_params *p)
-{
-	if (p->basic.logical_bs_shift < 9 || p->basic.logical_bs_shift > 12)
-		return false;
-	if (p->basic.logical_bs_shift > p->basic.physical_bs_shift)
-		return false;
-	return true;
-}
-
 static inline int ublk_queue_alloc_sqes(const struct ublksrv_queue *q,
 		struct io_uring_sqe *sqes[], int nr_sqes)
```

Comment:
ublk_param_is_valid() is actually generic, not related with specific target, so it belongs to ublksrv_tgt.h

Reply:
done